curvefs/src/client/inode_wrapper.h

```diff
@@ -48,52 +49,42 @@ enum InodeStatus {
     Error = -1,
 };
 
+
 // TODO(xuchaojie) : get from conf maybe?
 const uint32_t kOptimalIOBlockSize = 0x10000u;
 
 using rpcclient::MetaServerClient;
 using rpcclient::MetaServerClientImpl;
 
 std::ostream &operator<<(std::ostream &os, const struct stat &attr);
-void AppendS3ChunkInfoToMap(uint64_t chunkIndex, const S3ChunkInfo &info,
+void AppendS3ChunkInfoToMap(
```

Comment:
Too much formatting, almost nowhere to find actually modify place...

Reply:
done